spec/recipes/limits_spec.rb

```diff
@@ -18,20 +18,24 @@
 require_relative '../spec_helper'
```

Comment:
@ncs-alane good idea, lets do that

Still if `require 'spec_helper' is a more common way, we do not have anything special in the spec_helper, so that should be fine

Can you please add `--color` and `--format documentation` in the same step?